docs/resources/graph-optimizations.md

```diff
@@ -0,0 +1,154 @@
+---
+title: High level design
```

Comment:
title metadata should be Graph Optimizations

Reply:
Fixed